apps/nextjs-website/src/lib/tutorialListPages.ts

```diff
@@ -0,0 +1,48 @@
+import { mergeProductWithStaticContent } from './products';
+import { TutorialsPageProps } from '@/app/[productSlug]/tutorials/page';
+import { StrapiTutorialListPages } from './strapi/tutorialListCodec';
+import { Tutorial } from './types/tutorialData';
+
+type StaticTutorialListPages = readonly TutorialsPageProps[];
+
+export function makeTutorialListPagesProps(
+  strapiTutorialListPages: StrapiTutorialListPages,
+  staticTutorialListPages: StaticTutorialListPages
+): readonly TutorialsPageProps[] {
+  return [
+    ...strapiTutorialListPages.data.map(({ attributes }) => {
+      const product = mergeProductWithStaticContent(
+        attributes.product.data.attributes
+      );
+      const tutorials: readonly Tutorial[] = attributes.tutorials.data.map(
+        ({ attributes }) => ({
+          name: attributes.title,
+          path: `/${product.slug}/tutorials/${attributes.slug}`,
+          title: attributes.title,
+          publishedAt: attributes.publishedAt,
+          showInOverview: false,
```

Comment:
I think we don't need this property anymore

Reply:
I think `showInOverview` could be removed once static content is no loger used but for now i think is necessary